erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -360,3 +362,33 @@ def get_expense_claim(
 	)
 
 	return expense_claim
+
+# api to create expense claim
+@frappe.whitelist()
+def create_expense_claim(description, user_id, expense_date, expense_claim_type, amount):
```

Comment:
Also, change the parameters to 

def create_expense_claim(user, expense_date, expense_claim_type, description, amount):